eth/gasprice/feehistory.go

```diff
@@ -245,18 +245,18 @@ func (oracle *Oracle) FeeHistory(ctx context.Context, blocks int, unresolvedLast
 	}
 	oldestBlock := lastBlock + 1 - uint64(blocks)
 
-	// If pending txs are less than oracle.congestedThreshold, we consider the network to be non-congested and suggest
-	// a minimal tip cap. This is to prevent users from overpaying for gas when the network is not congested and a few
-	// high-priced txs are causing the suggested tip cap to be high.
+	// If the suggestedGasPrice equals to oracle.defaultGasTipCap or oracle.defaultBasePrice (before Curie (EIP-1559)),
+	// it means the latest block is out of capacity. We consider the network to be non-congested and suggest a minimal
+	// tip cap. This is to prevent users from overpaying for gas when the network is not congested and a few high-priced
+	// transactions are causing the suggested tip cap to be high.
```

Comment:
```suggestion
	// If the suggestedGasPrice equals to oracle.defaultGasTipCap or oracle.defaultBasePrice (before Curie (EIP-1559)),
	// it means the latest block is NOT out of capacity. We consider the network to be non-congested and suggest a minimal
	// tip cap. This is to prevent users from overpaying for gas when the network is not congested and a few high-priced
	// transactions are causing the suggested tip cap to be high.
```

Is there a `NOT` missing?

Reply:
Yeah, nice catch.